Modules/Scripted/DataProbe/Logic/vtkPVScalarBarActor.cxx

```diff
@@ -211,7 +211,7 @@ int vtkPVScalarBarActor::CreateLabel(
   // One Windows, formats with exponents have three digits by default
   // whereas on other systems, exponents have two digits. Set to two
   // digits on Windows for consistent behavior.
-#ifdef _WIN32
+#if _WIN32 && _MSC_VER < 1900
```

Comment:
Let's fix this more like was done in Paraview: https://github.com/Kitware/ParaView/commit/d4f2b29b4d63ae3dfd8553ee98a4d5eb181e8244